Configuration/AlCa/python/autoCond.py

```diff
@@ -20,9 +20,11 @@
     # GlobalTag for MC production (Heavy Ions collisions) with optimistic alignment and calibrations for Run2
     'run2_mc_hi'        :   '80X_mcRun2_HeavyIon_v3',
     # GlobalTag for Run1 data reprocessing
-    'run1_data'         :   '80X_dataRun2_v4',
+    'run1_data'         :   '80X_dataRun2_v5',
```

Comment:
git still shows it in the diffs.
I would expect a "revert 8879164d070ad " in the history of commits, not a plain commit that updates the file.